Tools/wasm/wasm_build.py

```diff
@@ -380,6 +472,12 @@ def getenv(self) -> dict:
         for key, value in platenv.items():
             if value is None:
                 env.pop(key, None)
+            elif key == "PATH":
+                # list of path items, prefix with extra paths
+                new_path: List[pathlib.PurePath] = []
+                new_path.extend(self.host.get_extra_paths())
+                new_path.extend(value)
+                env[key] = os.pathsep.join(os.fspath(p) for p in new_path)
```

Comment:
Just a suggestion; in no way important. 😄

```suggestion
                env[key] = os.pathsep.join(map(os.fspath, new_path))
```